### DISCUSSION TITLE
Session ID pysdk-c3d13b63c5 has an error during scheduling/startup or cancelled.

### DISCUSSION THREAD

**Discussion by yujung7768903** [lablup/contribution-academy-2021#111] [Q&A]
### Describe the problem
m1 관련해서 업데이트 된 후에 개발환경 재설치하고서 실행을 안해보고 있었는데, 실행을 해보니 에러 메세지가 뜹니다.

dev2103 개발 환경을 설치하고,아래의 명령어를 작성했습니다.
backend.ai run -c 'print("hello world")' python
그 후에 다음과 같은 에러 메세지가 떴습니다.
`✘ Session ID pysdk-c3d13b63c5 has an error during scheduling/startup or cancelled.`                 

<img width="701" alt="Screen Shot 2021-10-09 at 11 28 10 PM" src="https://user-images.githubusercontent.com/68562176/136662148-9d4817de-e0ec-4963-b662-34386f14b507.png">

* manager
<img width="1428" alt="Screen Shot 2021-10-13 at 3 25 32 PM" src="https://user-images.githubusercontent.com/68562176/137078617-e1c331ea-1d1e-4d23-a580-f70242a13655.png">

<img width="1426" alt="Screen Shot 2021-10-13 at 3 24 41 PM" src="https://user-images.githubusercontent.com/68562176/137078494-8ba8203f-8636-42fd-8f46-3cc0bd684314.png">
* agent
<img width="1430" alt="Screen Shot 2021-10-13 at 3 27 25 PM" src="https://user-images.githubusercontent.com/68562176/137078882-5a5f10e0-0835-40e7-ae65-211ed2a4416f.png">

`run.py`파일에서 관련된 부분을 찾기는 했는데, 왜 session status가 cancelled인지 왜 에러가 나는 것인지 모르겠습니다...
https://github.com/lablup/backend.ai-client-py/blob/main/src/ai/backend/client/cli/run.py#L526-L528
### Environment Description
OS: macOS Big Sur 11.2
App version: 21.03.5
Browser version: Chrome 91.0.4472.114
Python version: 2.7.16
Current Shell: zsh

**Comment by mindgitrwx** [comment]
실행하는 과정에서 agent 나 manager에서 어떤 log가 뜨는지 알 수 있으면 좋을 것 같습니다.

**Reply by yujung7768903** [reply]
아 그러쿤요! 방금 스크린샷 추가했습니다. 어느 부분을 캡쳐해야할지 모르겠어서 일단 반복적으로 뜨는 부분은 잘라서 올렸습니다.

**Reply by mindgitrwx** [reply]
agent에서 line 908에서 _handle_clean_event라고 로그에서 읽혀지는데, 가장 최근 버전의 agent Repo를 보면 861번 줄에 그 함수가 정의되어 있습니다.

**Reply by mindgitrwx** [reply]
가장 최근 버전의 agent repo를 인스톨하면 문제가 해결될 수도 있어 보입니다.
